e2e/test/scenarios/dashboard-filters/dashboard-filters-sql-number.cy.spec.js

```diff
@@ -58,15 +58,13 @@ describe("scenarios > dashboard > filters > SQL > text/category", () => {
     );
   });
 
-  it(`should work when set as the default filter`, () => {
+  it("should work when set as the default filter", () => {
     setFilter("Number", "Equal to");
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Default value").next().click();
+    cy.get("main").findByText("Default value").next().click();
```

Comment:
```suggestion
    sidebar().findByText("Default value").next().click();
```

also works, and is more explicit than just `cy.get("main")`